src/AccessibilityInsights.Extensions.GitHub/LinkValidator.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    /// <summary>
+    /// GitHub Repo Link Format Validator
+    /// </summary>
+    public static class LinkValidator
+    {
+        private static readonly string GitHubLink = Properties.Resources.GitHubLink;
+        private static readonly string AlphaNumricPattern = Properties.Resources.AlphaNumricPattern;
+
+        public static bool IsValidGitHubRepoLink(string Link)
```

Comment:
would it be a good idea to support both "www.github.com/Microsoft/ai-win" and "www.github.com/Microsoft/ai-win/issues". That way we are more robust. If they have /issue at the end we simply ignore it and go ahead with business as usual. We could also punt on that (according to the discussion today morning)

Reply:
I agree we need to support "www.GithUB" and "https://www.GitHub.com" but I don't think we need to handle "issues" as we agreed in the meeting, I remember the discussion was to keep it without issues and have the web team to remove it from the link

Reply:
Yes the discussion was to have the web team remove issues but we also said time permitting we should make it robust. In this case accepting the"/issues" at the end should be ignored whenever we get to making the application more robust. Not necessary as a part of this pr